datahub-web-react/src/app/entity/shared/components/styled/ExpandedActorGroup.tsx

```diff
@@ -1,18 +1,45 @@
+import { Popover, Typography } from 'antd';
 import React from 'react';
+import styled from 'styled-components';
 import { CorpGroup, CorpUser } from '../../../../../types.generated';
 import { ExpandedActor } from './ExpandedActor';
 
+const PopoverActors = styled.div``;
+
 type Props = {
     actors: Array<CorpUser | CorpGroup>;
+    max?: number | null;
     onClose?: (actor: CorpUser | CorpGroup) => void;
 };
 
-export const ExpandedActorGroup = ({ actors, onClose }: Props) => {
+const DEFAULT_MAX = 10;
+
+export const ExpandedActorGroup = ({ actors, max, onClose }: Props) => {
+    const finalMax = max || DEFAULT_MAX;
```

Comment:
nit: I think you can just set a default in this props destructuring to be like `({ actors, max = DEFAULT_MAX, onClose...`

Reply:
nice!